data/scripts.js

```diff
@@ -184,10 +184,6 @@ let BattleScripts = {
 
 		let attrs = '';
 
-		if (move.flags['charge'] && !pokemon.volatiles[move.id]) {
-			attrs = '|[still]'; // suppress the default move animation
-		}
-
```

Comment:
The `[still]` used to be manually added here, but that causes problems for two-turn moves that want to execute in one turn but have no target, which is why I removed it and added it manually to all the charge moves.

Reply:
Okay, neat.